return {
	{
		["Air Pressure"] = "1.00atm",
		["Base Car Weight"] = "25T",
		["Base Engine Weight"] = "50T",
		["Car Cargo Volume"] = "75m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$1,000K",
		["Description"] = "The Callisto is classic American steam locomotive, using immense mechanical forces to slowly pull massive amounts of cargo across the nation.",
		["Drag Coefficient"] = 0.8,
		["Friction Coefficient"] = 0.0035,
		["Fuel Cost Per Kilowatt"] = "$0.03",
		["Id"] = "Rail_Callisto",
		["Lifespan"] = "30yr",
		["Maintenance Cost"] = "$33K",
		["Manufacturer"] = "Lafayette Locomotive Works",
		["Max Car Cargo Weight"] = "100T",
		["Max Speed"] = "50 kph",
		["Max Speed Car Count"] = 16,
		["Model"] = "No. 119",
		["Name"] = "Callisto",
		["Vehicle Compatibility"] = "Rail",
	},
	{
		["Air Pressure"] = "1.00atm",
		["Base Car Weight"] = "25T",
		["Base Engine Weight"] = "100T",
		["Car Cargo Volume"] = "75m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$2,000K",
		["Description"] = "The Denver is an iconic American streamliner locomotive known for providing decades of reliable service.",
		["Drag Coefficient"] = 0.95,
		["Friction Coefficient"] = 0.0035,
		["Fuel Cost Per Kilowatt"] = "$0.27",
		["Id"] = "Rail_Denver",
		["Lifespan"] = "40yr",
		["Maintenance Cost"] = "$50K",
		["Manufacturer"] = "Monarch",
		["Max Car Cargo Weight"] = "100T",
		["Max Speed"] = "70 kph",
		["Max Speed Car Count"] = 30,
		["Model"] = "E7-F",
		["Name"] = "Denver",
		["Vehicle Compatibility"] = "Rail",
	},
	{
		["Air Pressure"] = "1.00atm",
		["Base Car Weight"] = "25T",
		["Base Engine Weight"] = "75T",
		["Car Cargo Volume"] = "75m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$2,500K",
		["Description"] = "The Pacifique is a world famous European HSR allowing for quick transportation between countries at affordable prices.",
		["Drag Coefficient"] = 0.3,
		["Friction Coefficient"] = 0.0035,
		["Fuel Cost Per Kilowatt"] = "$0.27",
		["Id"] = "HighSpeedRail_Pacifique",
		["Lifespan"] = "25yr",
		["Maintenance Cost"] = "$100K",
		["Manufacturer"] = "SNCR",
		["Max Car Cargo Weight"] = "100T",
		["Max Speed"] = "160 kph",
		["Max Speed Car Count"] = 12,
		["Model"] = "T-1988",
		["Name"] = "Pacifique",
		["Vehicle Compatibility"] = "HighSpeedRail",
	},
	{
		["Air Pressure"] = "1.00atm",
		["Base Car Weight"] = "25T",
		["Base Engine Weight"] = "75T",
		["Car Cargo Volume"] = "75m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$5,000K",
		["Description"] = "The Gulf is a retro-futuristic vision for urban transit, however it is also very capable in dealing with high incline mountainous regions due to how it grips the track.",
		["Drag Coefficient"] = 0.5,
		["Friction Coefficient"] = 0.00438,
		["Fuel Cost Per Kilowatt"] = "$0.20",
		["Id"] = "Monorail_Gulf",
		["Lifespan"] = "30yr",
		["Maintenance Cost"] = "$167K",
		["Manufacturer"] = "Walt Motors",
		["Max Car Cargo Weight"] = "100T",
		["Max Speed"] = "180 kph",
		["Max Speed Car Count"] = 8,
		["Model"] = "D71",
		["Name"] = "Gulf",
		["Vehicle Compatibility"] = "Monorail",
	},
	{
		["Air Pressure"] = "1.00atm",
		["Base Car Weight"] = "25T",
		["Base Engine Weight"] = "50T",
		["Car Cargo Volume"] = "75m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$5,000K",
		["Description"] = "The Magnus is a record-breaking Japanese maglev that serves as the backbone for fast transit between densely populated cities. The 'Magnus' branding is an English localization to help market the technology overseas.",
		["Drag Coefficient"] = 0.2,
		["Friction Coefficient"] = 0.00035,
		["Fuel Cost Per Kilowatt"] = "$0.07",
		["Id"] = "MaglevRail_Magnus",
		["Lifespan"] = "25yr",
		["Maintenance Cost"] = "$200K",
		["Manufacturer"] = "Magurebu",
		["Max Car Cargo Weight"] = "100T",
		["Max Speed"] = "600 kph",
		["Max Speed Car Count"] = 12,
		["Model"] = "TK46",
		["Name"] = "Magnus",
		["Vehicle Compatibility"] = "MaglevRail",
	},
	{
		["Air Pressure"] = "0.20atm",
		["Base Car Weight"] = "25T",
		["Base Engine Weight"] = "50T",
		["Car Cargo Volume"] = "75m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$10,000K",
		["Description"] = "The Gigapod is about speed above everything else, including fiscal responsibility.",
		["Drag Coefficient"] = 0.1,
		["Friction Coefficient"] = 0.00035,
		["Fuel Cost Per Kilowatt"] = "$0.03",
		["Id"] = "HyperloopRail_Gigapod",
		["Lifespan"] = "15yr",
		["Maintenance Cost"] = "$667K",
		["Manufacturer"] = "Sturgeon",
		["Max Car Cargo Weight"] = "100T",
		["Max Speed"] = "1,200 kph",
		["Max Speed Car Count"] = 12,
		["Model"] = "TT20-F",
		["Name"] = "Gigapod",
		["Vehicle Compatibility"] = "HyperloopRail",
	},
	{
		["Air Pressure"] = "1.00atm",
		["Base Car Weight"] = "18T",
		["Base Engine Weight"] = "12T",
		["Car Cargo Volume"] = "50m^3",
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = "$70K",
		["Description"] = "The Marauder is an 'Merican Truck made in 'Merica with a tough name to show other trucking companies that you broke the mold and are really cool and probably a rebel and you don't take orders from nobody.",
		["Drag Coefficient"] = 0.5,
		["Friction Coefficient"] = 0.0105,
		["Fuel Cost Per Kilowatt"] = "$0.27",
		["Id"] = "Road_Marauder",
		["Lifespan"] = "15yr",
		["Maintenance Cost"] = "$5K",
		["Manufacturer"] = "Revolvo",
		["Max Car Cargo Weight"] = "15T",
		["Max Speed"] = "90 kph",
		["Max Speed Car Count"] = 1,
		["Model"] = "FH",
		["Name"] = "Marauder",
		["Vehicle Compatibility"] = "Road",
	},
}
