return {
	["HighSpeedRail_Pacifique"] = {
		["Air Pressure"] = 1,
		["Base Car Weight"] = 25,
		["Base Engine Weight"] = 75,
		["Car Cargo Volume"] = 75,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 2500,
		["Description"] = "The Pacifique is a world famous European HSR allowing for quick transportation between countries at affordable prices.",
		["Drag Coefficient"] = 0.3,
		["Friction Coefficient"] = 0.0035,
		["Fuel Cost Per Kilowatt"] = 0.2699914015,
		["Id"] = "HighSpeedRail_Pacifique",
		["Lifespan"] = 25,
		["Maintenance Cost"] = 100,
		["Manufacturer"] = "SNCR",
		["Max Car Cargo Weight"] = 100,
		["Max Speed"] = 160,
		["Max Speed Car Count"] = 12,
		["Model"] = "T-1988",
		["Name"] = "Pacifique",
		["Vehicle Compatibility"] = "HighSpeedRail",
	},
	["HyperloopRail_Gigapod"] = {
		["Air Pressure"] = 0.2,
		["Base Car Weight"] = 25,
		["Base Engine Weight"] = 50,
		["Car Cargo Volume"] = 75,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 10000,
		["Description"] = "The Gigapod is about speed above everything else, including fiscal responsibility.",
		["Drag Coefficient"] = 0.1,
		["Friction Coefficient"] = 0.00035,
		["Fuel Cost Per Kilowatt"] = 0.03374892519,
		["Id"] = "HyperloopRail_Gigapod",
		["Lifespan"] = 15,
		["Maintenance Cost"] = 666.6666667,
		["Manufacturer"] = "Sturgeon",
		["Max Car Cargo Weight"] = 100,
		["Max Speed"] = 1200,
		["Max Speed Car Count"] = 12,
		["Model"] = "TT20-F",
		["Name"] = "Gigapod",
		["Vehicle Compatibility"] = "HyperloopRail",
	},
	["MaglevRail_Magnus"] = {
		["Air Pressure"] = 1,
		["Base Car Weight"] = 25,
		["Base Engine Weight"] = 50,
		["Car Cargo Volume"] = 75,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 5000,
		["Description"] = "The Magnus is a record-breaking Japanese maglev that serves as the backbone for fast transit between densely populated cities. The 'Magnus' branding is an English localization to help market the technology overseas.",
		["Drag Coefficient"] = 0.2,
		["Friction Coefficient"] = 0.00035,
		["Fuel Cost Per Kilowatt"] = 0.06749785039,
		["Id"] = "MaglevRail_Magnus",
		["Lifespan"] = 25,
		["Maintenance Cost"] = 200,
		["Manufacturer"] = "Magurebu",
		["Max Car Cargo Weight"] = 100,
		["Max Speed"] = 600,
		["Max Speed Car Count"] = 12,
		["Model"] = "TK46",
		["Name"] = "Magnus",
		["Vehicle Compatibility"] = "MaglevRail",
	},
	["Monorail_Gulf"] = {
		["Air Pressure"] = 1,
		["Base Car Weight"] = 25,
		["Base Engine Weight"] = 75,
		["Car Cargo Volume"] = 75,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 5000,
		["Description"] = "The Gulf is a retro-futuristic vision for urban transit, however it is also very capable in dealing with high incline mountainous regions due to how it grips the track.",
		["Drag Coefficient"] = 0.5,
		["Friction Coefficient"] = 0.004375,
		["Fuel Cost Per Kilowatt"] = 0.2024935512,
		["Id"] = "Monorail_Gulf",
		["Lifespan"] = 30,
		["Maintenance Cost"] = 166.6666667,
		["Manufacturer"] = "Walt Motors",
		["Max Car Cargo Weight"] = 100,
		["Max Speed"] = 180,
		["Max Speed Car Count"] = 8,
		["Model"] = "D71",
		["Name"] = "Gulf",
		["Vehicle Compatibility"] = "Monorail",
	},
	["Rail_Callisto"] = {
		["Air Pressure"] = 1,
		["Base Car Weight"] = 25,
		["Base Engine Weight"] = 50,
		["Car Cargo Volume"] = 75,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 1000,
		["Description"] = "The Callisto is classic American steam locomotive, using immense mechanical forces to slowly pull massive amounts of cargo across the nation.",
		["Drag Coefficient"] = 0.8,
		["Friction Coefficient"] = 0.0035,
		["Fuel Cost Per Kilowatt"] = 0.03439380911,
		["Id"] = "Rail_Callisto",
		["Lifespan"] = 30,
		["Maintenance Cost"] = 33.33333333,
		["Manufacturer"] = "Lafayette Locomotive Works",
		["Max Car Cargo Weight"] = 100,
		["Max Speed"] = 50,
		["Max Speed Car Count"] = 16,
		["Model"] = "No. 119",
		["Name"] = "Callisto",
		["Vehicle Compatibility"] = "Rail",
	},
	["Rail_Denver"] = {
		["Air Pressure"] = 1,
		["Base Car Weight"] = 25,
		["Base Engine Weight"] = 100,
		["Car Cargo Volume"] = 75,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 2000,
		["Description"] = "The Denver is an iconic American streamliner locomotive known for providing decades of reliable service.",
		["Drag Coefficient"] = 0.95,
		["Friction Coefficient"] = 0.0035,
		["Fuel Cost Per Kilowatt"] = 0.2699914015,
		["Id"] = "Rail_Denver",
		["Lifespan"] = 40,
		["Maintenance Cost"] = 50,
		["Manufacturer"] = "Monarch",
		["Max Car Cargo Weight"] = 100,
		["Max Speed"] = 70,
		["Max Speed Car Count"] = 30,
		["Model"] = "E7-F",
		["Name"] = "Denver",
		["Vehicle Compatibility"] = "Rail",
	},
	["Road_Marauder"] = {
		["Air Pressure"] = 1,
		["Base Car Weight"] = 17.5,
		["Base Engine Weight"] = 12,
		["Car Cargo Volume"] = 50,
		["Car Passenger Limit"] = 0,
		["Construction Cost"] = 70,
		["Description"] = "The Marauder is an 'Merican Truck made in 'Merica with a tough name to show other trucking companies that you broke the mold and are really cool and probably a rebel and you don't take orders from nobody.",
		["Drag Coefficient"] = 0.5,
		["Friction Coefficient"] = 0.0105,
		["Fuel Cost Per Kilowatt"] = 0.2699914015,
		["Id"] = "Road_Marauder",
		["Lifespan"] = 15,
		["Maintenance Cost"] = 4.666666667,
		["Manufacturer"] = "Revolvo",
		["Max Car Cargo Weight"] = 15,
		["Max Speed"] = 90,
		["Max Speed Car Count"] = 1,
		["Model"] = "FH",
		["Name"] = "Marauder",
		["Vehicle Compatibility"] = "Road",
	},
}
